### ISSUE TITLE
IllegalStateException: Session …~ROOT#opencast already in cache

### ISSUE THREAD

**Issue by lkiesow** [opencast/opencast#4971]
In Opencast 14 Jetty regularly throws exceptions complaining about sessions already being cached:

```
2023-05-26T14:39:50,895 | WARN  | (JettyAwareLogger:607) - 
java.lang.IllegalStateException: Session node01ent3agx9i0ss1h27bp1p88xxf111380~ROOT#opencast already in cache
	at org.eclipse.jetty.server.session.AbstractSessionCache.add(AbstractSessionCache.java:467) ~[!/:9.4.50.v20221201]
	at org.eclipse.jetty.server.session.SessionHandler.newHttpSession(SessionHandler.java:770) ~[!/:9.4.50.v20221201]
	at org.eclipse.jetty.server.Request.getSession(Request.java:1630) ~[!/:9.4.50.v20221201]
```

Full log:
[opencast.log](https://github.com/opencast/opencast/files/11576303/opencast.log)

**Comment by lkiesow** [comment]
Fixed by #5050

**Comment by DZenker** [comment]
I've just upgraded our production system from 12.x to 14.2 (which should include #5050). Now I have this issue, too. Furthermore, apart from regular error messages, **the whole system is completely dysfunctional!** Thus, I had to downgrade to 12.x.

Obviously, fix #5050 doesn't work in all settings. Are there our ways to solve this problem?

**Comment by DZenker** [comment]
Part of the log attached.
[opencast.log](https://github.com/opencast/opencast/files/12394261/opencast.log)

**Comment by gregorydlogan** [comment]
Hi @DZenker, can you send the full log to make sure there isn't something *else* wrong?  The error you attached shouldn't completely break the system, so I'm guessing it's something else earlier on in the boot process.  There are adopters on 14 already who *aren't* seeing this, and that suggests to me that it's something related to this specific system.

**Comment by DZenker** [comment]
Hi @gregorydlogan ,

the "broken system" was caused by another issue that I could solve. Indeed, the "session..." error did not have any obvious impact on the system. 

But, unfortunately I cannot investigate this issue further since I had to downgrade our system back to 12.x due to other issues, in particular https://github.com/opencast/opencast/issues/5184 resp. https://github.com/opencast/opencast/issues/5016 and the missing theodul player redirect in OC 14.
